taskw/test/test_datas.py

```diff
@@ -143,29 +143,17 @@ def test_updating_task(self):
 
         tasks = self.tw.load_tasks()
         assert len(tasks['pending']) == 1
-        assert tasks['pending'][0]['priority'] == 'L'
 
-        # For compatibility with the direct and shellout modes.
-        # Shellout returns more information.
-        try:
-            # Shellout mode returns the correct urgency, so,
-            # let's just not compare for now.
-            del tasks['pending'][0]['urgency']
-            del task['urgency']
+        updated_task = tasks['pending'][0]
+        assert updated_task['priority'] == 'L'
 
-            # Also, experimental mode returns the id.  So, avoid comparing.
-            del tasks['pending'][0]['id']
+        calculated_fields = ['urgency', 'id', 'modified']
```

Comment:
I debated whether to use field metadata here, too, and eventually landed on _not_ using it so the test doesn't need to be aware of where field metadata is stored.